src/functions/New-PuppetDscModule.ps1

```diff
@@ -103,6 +103,10 @@ Function New-PuppetDscModule {
           }
           Out-Utf8File -Path $PuppetProviderFilePath -InputObject $Resource.Provider
         }
+        
+        # Generate REFERENCE.md file for the Puppet module from the auto-generated types for each DSC resource
+        Set-PSModulePath -Path $InitialPsModulePath
```

Comment:
I think this line can be removed as it gets called in the `finally` block on line 119

Reply:
@michaeltlombardi the pdk command was failing if it's not set. SO added the Set-PSModulePath